app/javascript/react/components/MonsterForm.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import { Redirect } from 'react-router-dom'
+
+const MonsterForm = () => {
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+
+  const [formData, setFormData] = useState({
+    name: "",    description: "",
+    classification: "",
+    habitat: "",
+  })
+  const[postResponse, setPostResponse] = useState("")
+
+  const handleChange = (event) => {
+    setFormData({
+      ...formData,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  }
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    postNewMonster()
+  }
+  const postNewMonster = async () => {
+    try { 
+      const response = await fetch("/api/v1/monsters", {
+        method: "POST",
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({monster: formData})
+      })
+      if(!response.ok) {
+        throw new Error(`${response.status} (${response.statusText})`)
+      }
+      const parsedResponse = await response.json()
+      setPostResponse(parsedResponse.response)
+      setShouldRedirect(true) 
```

Comment:
Since we are setting redirect to true and navigating off this page, we don't need to update the response state on 37 and can remove that line.